package.json

```diff
@@ -25,7 +25,8 @@
     "react": "15.4.2"
   },
   "dependencies": {
-    "lodash": "^4.17.4"
+    "lodash": "^4.17.4",
```

Comment:
prelude-ls doesn't have a couple lodash methods we need. With lodash we are only pulling in the modules we need instead of the whole thing. prelude-ls does look tiny compared to lodash but there are plenty of methods it doesn't have